collector/serverstats.go

```diff
@@ -135,6 +135,66 @@ var (
 		),
 		prometheus.GaugeValue,
 	}
+
+	serverstatsNTPDaemonRxTimestamps = typedDesc{
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, serverstatsSubsystem, "ntp_daemon_rx_timestamps"),
```

Comment:
Sorry, one more nit, `prometheus.CounterValue` metric names must end in `_total`.

```suggestion
			prometheus.BuildFQName(namespace, serverstatsSubsystem, "ntp_daemon_rx_timestamps_total"),
```

Reply:
Done.